plugins/design-system/pages/ui-components/banners/tabs/OverviewTab.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from "react";
+
+class OverviewTab extends Component {
+  render() {
+    return (
```

Comment:
Can we extract some parts to components so that way we can re use them.

example: `DesignSystemTable.js` `DesignSystemDescription.js` and etc.

What you think @jfurrow